### PR TITLE
Re-adds black cowboy hat to loadout

### PR DESCRIPTION
**Pull request by AffectedArc07**
This will likely be my only PR and thats because I was asked nicely for it

---

## About The Pull Request
I got asked to re-add the black cowboy hat as a loadout item because the colourable ones just dont look right for the character.

This is beauty
image

This is bad
image

## How This Contributes To The Skyrat Roleplay Experience
Lets someone play a character how they want

## Proof of Testing
image

## Changelog

:cl: AffectedArc07
add: Re-adds black cowboy hat to the loadout
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 1]
```diff
@@ -336,3 +336,12 @@
 	greyscale_config_worn_teshari = /datum/greyscale_config/standalone_hood/worn/teshari
 	greyscale_config_worn_better_vox = /datum/greyscale_config/standalone_hood/worn/newvox
 	greyscale_config_worn_vox = /datum/greyscale_config/standalone_hood/worn/oldvox
+
+/obj/item/clothing/head/costume/cowboyhat_old
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	name = "old cowboy hat"
+	desc = "An older cowboy hat, perfect for any outlaw, though lacking fancy colour magic"
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm]:
```suggestion
	desc = "An older cowboy hat, perfect for any outlaw, though lacking fancy colour magic."
```

**Comment by ghost** [comment]
Re-adding an old and outdated sprite just to please somebody feels kind of... wrong.
I'm entirely in support of respriting the recolorable hat to have the curve back, raising it up a pixel and darkening it. I messed that sprite up a lot, honestly... 
But having 'item' and 'item but old' is kind of pointless.

**Comment by AffectedArc07** [comment]
> Re-adding an old and outdated sprite just to please somebody feels kind of... wrong. I'm entirely in support of respriting the recolorable hat to have the curve back, raising it up a pixel and darkening it. I messed that sprite up a lot, honestly... But having 'item' and 'item but old' is kind of pointless.

If the new item wasn't worse than the old then the old wouldn't need to be revived. 
image
This does not look like a cowboy hat. Theres no shading and it looks far too flat.

**Comment by ghost** [comment]
> If the new item wasn't worse than the old then the old wouldn't need to be revived. image This does not look like a cowboy hat. Theres no shading and it looks far too flat.

**"I'm entirely in support of respriting the recolorable hat to have the curve back, raising it up a pixel and darkening it"**

**Comment by tf-4** [comment]
> > If the new item wasn't worse than the old then the old wouldn't need to be revived. image This does not look like a cowboy hat. Theres no shading and it looks far too flat.
> 
> **"I'm entirely in support of respriting the recolorable hat to have the curve back, raising it up a pixel and darkening it"**

feel free to do this as a replacement in a follow up PR but for the sake of keeping things moving I'll go ahead with this one for the time being

**Review by tf-4 — APPROVED** [review]

**Pull request merged**